test/e2e/page-objects/pages/dialog/snap-install.ts

```diff
@@ -0,0 +1,57 @@
+import { Driver } from '../../../webdriver/driver';
+
+class SnapInstall {
+  private driver: Driver;
+
+  private readonly nextPageButton =
+    '[data-testid="page-container-footer-next"]';
+
+  private readonly pageFooter = '.page-container__footer';
+
+  private readonly permissionConnect = '.permissions-connect';
+
+  private readonly scrollSnapInstall = '[data-testid="snap-install-scroll"]';
+
+  private readonly approveButton = '[data-testid="confirmation-submit-button"]';
+
+  constructor(driver: Driver) {
+    this.driver = driver;
+  }
+
+  async check_pageIsLoaded(): Promise<void> {
+    try {
+      await this.driver.waitForMultipleSelectors([
+        this.pageFooter,
+        this.permissionConnect,
+      ]);
+    } catch (e) {
+      console.log(
+        'Timeout while waiting for Snap install dialog to be loaded',
+        e,
+      );
+      throw e;
+    }
+    console.log('Snap install dialog is loaded');
+  }
+
+  async clickNextButton() {
+    console.log('Wait and click Confirm/Ok button');
+    await this.driver.waitForSelector(this.nextPageButton);
+    await this.driver.clickElement(this.nextPageButton);
```

Comment:
Nit: since within `clickElement` we have selector `findClickableElement`, and for this selector and `waitForSelector ` and they all use `elementLocated`, i believe we can remove `waitForSelector`. 
```suggestion
    await this.driver.clickElement(this.nextPageButton);
```

Reply:
That makes sense, @DDDDDanica. The reason I added it is because it's a Flask build and hence was needed. However, after your comment I have updated it in other files as well and executed it locally. It worked fine in both Chrome and Firefox browsers. I will monitor it in the CI for any flakiness too

Reply:
Sounds good. Thanks Harika !